src/extensions/php_fpm_metrics_test.go

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) F5, Inc.
+ *
+ * This source code is licensed under the Apache License, Version 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package extensions
+
+import (
+	"testing"
+
+	"github.com/nginx/agent/v2/src/core/config"
+	tutils "github.com/nginx/agent/v2/test/utils"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPhpFpmMetrics(t *testing.T) {
+	type test struct {
+		name string
+	}
+	tests := []test{
+		{
+			name: "intialize php-fpm metrics",
+		},
+	}
+
+	env := tutils.GetMockEnv()
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			_, err := NewPhpFpmMetrics(env, &config.Config{})
+			assert.NoError(t, err)
+		})
+	}
+}
```

Comment:
do we have other test cases in mind here? If not recommend dropping the table-driven boilerplate

```suggestion
func TestNewPhpFpmMetrics(t *testing.T) {
	_, err := NewPhpFpmMetrics(env, &config.Config{})
	assert.NoError(t, err)
}
```

Reply:
Yes. I plan to add more test cases as we extend on the code, so I prefer keeping it.